fuel-client/src/lib.rs

```diff
@@ -1,2 +1,10 @@
 pub mod client;
 pub mod schema;
+
+// re-exports
```

Comment:
Can you also document this in the PR description? Will be useful!

Reply:
added a note